provider-ui/pages/_document.js

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Document, { Head, Main, NextScript } from 'next/document';
+import Document, { Head, Main, NextScript, Html } from 'next/document';
 import flush from 'styled-jsx/server';
 
 class MesheryDocument extends Document {
   render() {
     // eslint-disable-next-line no-unused-vars
     const { pageContext } = this.props;
     return (
-      <html lang="en" dir="ltr">
+      <Html lang="en" dir="ltr">
```

Comment:
Interesting. Is the Capitalization significant?

Reply:
Html is the next.js document wrappers and html is what plainular html means. Html has some added meta tags enhanced for the next

Reply:
Oh, I see. Hmpf.